kura/org.eclipse.kura.wire.component.provider/OSGI-INF/metatype/org.eclipse.kura.wire.CloudSubscriber.xml

```diff
@@ -20,6 +20,26 @@
             default="(kura.service.pid=changeme)"
             description="Specifies, as an OSGi target filter, the pid of the Cloud Subscriber used to receive messages from the cloud platform.">
         </AD>
+        
+        <AD id="set.property.from.body"
+            name="Set property from message body"
+            type="String"
+            cardinality="0"
+            required="false"
+            default=""
+            description="If set to a non empty value, the publisher will add a metric to the emitted envelope containing with the content of the body of received message. The name of the emitted property is the value of this parameter, its type depends on the Body property type parameter">
```

Comment:
> If set to a non empty value, the publisher will add a metric to the emitted envelope containing with the content of the body of received message. The name of the emitted property is the value of this parameter, its type depends on the Body property type parameter

If set to a non empty value, the publisher will add a metric to the emitted envelope containing the received message body content. The name of the emitted property is the value of this parameter, its type depends on the Body property type parameter